b/babel-plugin-transform-runtime/babel-plugin-transform-runtime_ubi8_3.sh

```diff
@@ -0,0 +1,44 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : babel-plugin-transform-runtime
+# Version       : v7.4.0
+# Source repo   : https://github.com/babel/babel
+# Tested on     : UBI 8.3
+# Script License: Apache-2.0 License    
+# Maintainer    : Varsha Aaynure <Varsha.Aaynure@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+#Variables
+PACKAGE_URL=https://github.com/babel/babel.git
+PACKAGE_VERSION="${1:-v7.4.0}"
+
+yum update -y 
```

Comment:
Please avoid `yum update`.
Do it only if it is required.

Reply:
okay.